test/Sema/accessibility_shared_private.swift

```diff
@@ -44,13 +51,14 @@ extension Outer.Middle.Inner {
   }
 }
 
+// Padded to start on line 55 to make @LINE-55 work
```

Comment:
You could avoid this by using the split version as the input to FileCheck (and running FileCheck twice for the WMO case).